var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -55,31 +59,31 @@ class Caliper(CMakePackage):
             description='Enable SOSflow support')
     variant('fortran', default=False,
             description='Enable Fortran support')
+    variant('cuda', default=False,
```

Comment:
Sorry, I didn't see this is the first round of reviews. It's better to extend CudaPackage, that way you get all the conflicts for unsupported compilers for free, and you don't have to define the variant. Would that make sense for this package?

Reply:
We're not strictly a CUDA program, we only use support libraries that ship with it (nvtx and cupti). Still, it seems to simplify things so I've made Caliper a CudaPackage now.